cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -98,6 +110,42 @@ public static void validateConfig(SecureStore secureStore, SourceControlConfig s
     }
   }
 
+  /**
+   * Returns the Git file hash of the requested file path in the current HEAD.
```

Comment:
Please link to the git hash documentation

Reply:
Added.